src/atoms/cc-input-number.js

```diff
@@ -201,11 +230,31 @@ export class CcInputNumber extends LitElement {
         }
 
         label {
+          align-items: flex-end;
           cursor: pointer;
-          display: block;
+          display: flex;
+          gap: 2em;
+          justify-content: space-between;
           padding-bottom: 0.35em;
         }
 
+        .required {
+          color: var(--color-text-light);
+          font-size: 0.9em;
+          font-variant: small-caps;
+        }
+
+        slot[name='help']::slotted(*) {
+          color: var(--color-text-light);
+          font-size: 0.9em;
+          margin: 0.3em 0 0 0;
+        }
+        
+        slot[name='error']::slotted(*) {
+          color: var(--color-text-danger);
+          margin: 0.5em 0 0 0;
+        }
+
```

Comment:
all the CSS from `label` to here is exactly the same for the 3 form controls (text, number and select). Should we move this into a shared style ?
To me, these elements should always look the same across these 3 form controls so I don't see why not.

Reply:
I would say no for now but maybe add a comment on each 3 form controls. This way we know it's duplicated and we know we could address it when it's the right time.